bitnami/kafka/templates/broker/configmap.yaml

```
@@ -40,6 +40,8 @@ data:
     inter.broker.protocol.version={{ default (regexFind "^[0-9].[0-9]+" .Chart.AppVersion) .Values.interBrokerProtocolVersion }}
     {{- end }}
     {{- end }}
+    log.dir={{ printf "%s/data" .Values.broker.persistence.mountPath }}
+    logs.dir={{ .Values.broker.logPersistence.mountPath }}
```

Could you please add a comment to help differentiate between both settings?

```suggestion
    # Kafka data logs directory
    log.dir={{ printf "%s/data" .Values.broker.persistence.mountPath }}
    # Kafka application logs directory
    logs.dir={{ .Values.broker.logPersistence.mountPath }}
```

---

done at https://github.com/bitnami/charts/pull/20476/commits/e670035011b154d8cc9ce843fb095b787ca49b26